applications/nrf_desktop/README.rst

```diff
@@ -817,17 +817,34 @@ Adding nRF21540 EK shield support
 The nRF Desktop application can be used with the :ref:`ug_radio_fem_nrf21540ek` shield, an RF front-end module (FEM) for the 2.4 GHz range extension.
 The shield can be used with any nRF Desktop HID application configured for a development kit that is fitted with Arduino-compatible connector (see the :guilabel:`DK` tab in `Requirements`_).
 This means that the shield support is not available for nRF Desktop's dedicated boards, such as ``nrf52840gmouse_nrf52840``, ``nrf52kbd_nrf52832``, or ``nrf52840dongle_nrf52840``.
+
+Low Latency Packet mode
+-----------------------
+
+The RF front-end module (FEM) cannot be used together with Low Latency Packet Mode (LLPM) due to timing requirements.
+The LLPM support in the nRF Desktop application (:kconfig:option:`CONFIG_CAF_BLE_USE_LLPM`) must be disabled for builds with FEM.
+
+Building with EK shield support
+-------------------------------
+
 To build the application with the shield support, pass the ``SHIELD`` parameter to the build command.
+Make sure to also disable the LLPM support.
+For example, you can build the application for ``nrf52840dk_nrf52840`` with ``nrf21540ek`` shield using the following command:
 
-For detailed information about building the nRF Desktop application for the nRF21540 EK, see the :ref:`ug_radio_fem_nrf21540ek_programming` section on the nRF21540 EK documentation page.
+.. code-block:: console
 
-.. note::
-   For the multi-core build, use the ``hci_rpmsg_`` as the *childImageName* parameter, because in the nRF Desktop application, network core runs using ``hci_rpmsg_``.
-   The command would look as follows:
+   west build -b nrf52840dk_nrf52840 -- -DSHIELD=nrf21540ek -DCONFIG_CAF_BLE_USE_LLPM=n
 
-   .. code-block:: console
+For the multi-core build, you need to pass the ``SHIELD`` parameter to images built on both application and network core.
+The network core controls the FEM, but the application core needs to forward the needed pins to the network core.
+Use the ``hci_rpmsg_`` as the *childImageName* parameter, because in the nRF Desktop application, network core runs using ``hci_rpmsg_``.
```

Comment:
```suggestion
Use ``hci_rpmsg_`` as the *childImageName* parameter, because in the nRF Desktop application, network core runs using ``hci_rpmsg_``.
```